### PR TITLE
Add service module for ssh proxy example

### PR DESCRIPTION
**Pull request by pass-by-value**

### CODE REVIEW

File: salt/modules/ssh_service.py [hunk 1]
```diff
@@ -0,0 +1,136 @@
+# -*- coding: utf-8 -*-
+'''
+Provide the service module for the proxy-minion SSH sample
+'''
```
Comment (jfindlay) [inline salt/modules/ssh_service.py]: @pass-by-value, will you add a `.. versionadded:: 2015.8.2` here?  Thanks.

Comment (pass-by-value) [inline salt/modules/ssh_service.py]: Will do, could you please hold off the merge until then?

**Comment by jfindlay** [comment]
Sure, thanks @pass-by-value.

**Comment by cachedout** [comment]
@pass-by-value Please let me know when this one is ready to go.

**Comment by pass-by-value** [comment]
@cachedout Updated

**Comment by cachedout** [comment]
@pass-by-value Thanks!

**Pull request merged**